inc/tasktemplateinjection.class.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/*
+ * @version $Id: HEADER 14684 2011-06-11 06:32:40Z remi $
+ LICENSE
+
+ This file is part of the datainjection plugin.
+
+ Datainjection plugin is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+
+ Datainjection plugin is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with datainjection. If not, see <http://www.gnu.org/licenses/>.
+ --------------------------------------------------------------------------
+ @package   datainjection
+ @author    the datainjection plugin team
+ @copyright Copyright (c) 2010-2017 Datainjection plugin team
+ @license   GPLv2+
+            http://www.gnu.org/licenses/gpl.txt
+ @link      https://github.com/pluginsGLPI/datainjection
+ @link      http://www.glpi-project.org/
+ @since     2009
+ ---------------------------------------------------------------------- */
+
+if (!defined('GLPI_ROOT')) {
+    die("Sorry. You can't access directly to this file");
+}
+
+class PluginDatainjectionTaskTemplateInjection extends TaskTemplate
+                                        implements PluginDatainjectionInjectionInterface
+{
+
+
+   static function getTable($classname = null) {
+
+      $parenttype = get_parent_class();
+      return $parenttype::getTable();
+   }
+
+
+   function isPrimaryType() {
+
+      return true;
+   }
+
+
+   function connectedTo() {
+
+      return [];
+   }
+
+
+    /**
+    * @see plugins/datainjection/inc/PluginDatainjectionInjectionInterface::getOptions()
+   **/
+   function getOptions($primary_type = '') {
+
+      $tab           = Search::getOptions(get_parent_class($this));
+
+      //Remove some options because some fields cannot be imported
+      $blacklist     = PluginDatainjectionCommonInjectionLib::getBlacklistedOptions(get_parent_class($this));
+      $notimportable = [];
+
+      $options['ignore_fields'] = array_merge($blacklist, $notimportable);
+
+      $options['displaytype']   = ["bool"           => [86],
+                                      "multiline_text" => [16]];
```

Comment:
```suggestion
                                      "multiline_text" => [4, 16]];
```

The "Content"  field of the task template is a multi line text and without this option will be limited to 255 chars.